docs/setup_bothan.md

```diff
@@ -0,0 +1,52 @@
+# Bothan Installation Guide
+
+## Prerequisites
+
+Before building and running Bothan, ensure the following dependencies are installed:
+
+- [Clang](https://clang.llvm.org/get_started.html)
+- [Rust](https://www.rust-lang.org/tools/install)
+
+## Building Bothan
+
+To install the Bothan CLI, follow the steps below:
+
+- Clone or navigate to the bothan-api repository.
+- Run the following command to build and install the Bothan CLI
+  binary:
+
+```bash 
+cargo install --path bothan-api/server-cli --bin bothan
+```
+
+This will compile and install the bothan executable
```

Comment:
Add more ways to install binary (install/build/download) (example: https://hermes.informal.systems/quick-start/installation.html). can do it in another PR.

Reply:
noted